superset/config.py

```diff
@@ -571,10 +572,6 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 SMTP_PASSWORD = "superset"
 SMTP_MAIL_FROM = "superset@superset.com"
 
-if not CACHE_DEFAULT_TIMEOUT:
```

Comment:
This logic is in the same block as line #313 and always evaluates to `True` and thus isn't required (hence the deletion).